C/Calcium/build_tarballs.jl

```diff
@@ -61,8 +61,8 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency(PackageSpec(name="FLINT_jll"), compat = "~200.700"),
-    Dependency(PackageSpec(name="Arb_jll"), compat = "~200.1900"),
+    Dependency(PackageSpec(name="FLINT_jll"), compat = "~200.800"),
+    Dependency(PackageSpec(name="Arb_jll"), compat = "~200.2000"),
     Dependency(PackageSpec(name="Antic_jll"), compat = "~0.200"),
```

Comment:
I think this should be either
```suggestion
    Dependency(PackageSpec(name="Antic_jll"); compat = "~0.200.300"),
```
or
```suggestion
    Dependency(PackageSpec(name="Antic_jll"), v"0.200.300"; compat = "~0.200"),
```
so that the build version is an actually existing version